src/Link.js

```diff
@@ -17,31 +16,26 @@ const buttonStyles = {
   appearance: 'none'
 }
 
-const hoverColor = system({
-  hoverColor: {
-    property: 'color',
-    scale: 'colors'
-  }
-})
-
-const Link = styled.a`
+const Link = styled.a.attrs(props => ({
+  color: props.color ? props.color : props.muted ? 'gray.6' : 'blue.5'
+}))`
   text-decoration: ${props => (props.underline ? 'underline' : 'none')};
   &:hover {
-    text-decoration: underline;
-    ${hoverColor};
+    text-decoration: ${props => (props.muted ? 'none' : 'underline')};
+    ${props => (props.muted ? 'color: ' + get('colors.blue.5')(theme) : '')};
```

Comment:
I'm surprised the linter didn't catch this but can you use a tagged template literal here instead?
```suggestion
    ${props => (props.muted ? `color: ${get('colors.blue.5')(theme)}`: '')};
```